client/cli/src/runner.rs

```diff
@@ -153,7 +153,7 @@ impl<C: SubstrateCli> Runner<C> {
 	/// 2020-06-03 16:14:21 💾 Database: RocksDb at /tmp/c/chains/flamingfir7/db
 	/// 2020-06-03 16:14:21 ⛓  Native runtime: node-251 (substrate-node-1.tx1.au10)
 	/// ```
-	pub fn print_node_infos(&self, runtime_version: RuntimeVersion) {
+	fn print_node_infos(&self, runtime_version: RuntimeVersion) {
```

Comment:
I wonder if that `runtime_version` couldn't be move to SubstrateCli since it's a kind of constant. This would simplifies a bit the calls here.

Reply:
Done that now

Reply:
That's amazing. It looks neat!